Kontent.Ai.Delivery.Caching/DistributedCacheManager.cs

```diff
@@ -15,25 +16,38 @@ internal class DistributedCacheManager : IDeliveryCacheManager
     {
         private readonly IDistributedCache _distributedCache;
         private readonly DeliveryCacheOptions _cacheOptions;
+        private readonly ILogger _logger;
 
         /// <summary>
         /// Initializes a new instance of <see cref="DistributedCacheManager"/>
         /// </summary>
         /// <param name="distributedCache">An instance of an object that represent distributed cache</param>
         /// <param name="cacheOptions">The settings of the cache</param>
-        public DistributedCacheManager(IDistributedCache distributedCache, IOptions<DeliveryCacheOptions> cacheOptions)
+        /// <param name="loggerFactory">The factory used to create loggers.</param>
+        public DistributedCacheManager(IDistributedCache distributedCache,
+            IOptions<DeliveryCacheOptions> cacheOptions,
+            ILoggerFactory loggerFactory)
```

Comment:
Is it possible to make this optional? People could choose to opt-out then. Also, it would be more friendly from a backward compatibility point of view.

Reply:
Oh, I see, that you made it optional in `AddDeliveryClientCache`, but is this ctor accessible from the customer project as well?

Reply:
I thought about it. It seems, that this ctor is available from the customer project. Nevertheless, in documentation we provide examples, which use:
- `CacheManagerFactory.Create()` method when customer registers multiple clients
- `AddDeliveryClientCache()` in ServiceCollectionExtensions, when customer registers a single client
Both these methods create entity of `DistributedCacheManager` class and have `loggerFactory` as optional parameter.
I don't know, whether some of our clients can use this ctor directly

Reply:
Finally, I decided to make it optional in this ctor too. Just to be sure that I will not break someone's code